static/js/vue-cdr-access/src/components/browseSearch.vue

```diff
@@ -78,16 +78,16 @@ Search form component displayed on full record pages, allowing keyword searches
 
         methods: {
             getResults() {
-                let update_params = { anywhere: encodeURIComponent(this.search_query) };
+                let update_params = { anywhere: encodeURIComponent(this.search_query), start: 0 };
                 this.routeWithParams(this.urlParams(update_params, 'displayRecords'));
```

Comment:
if you add it, it would be good to use `resetStartRow` whenever possible for resetting the start row in param objects

Reply:
I had that at one point, but took it out.